File: 30 Days of Code/Java/Day8_Dictionary_Maps.java
//Complete this code or write your own from scratch
import java.util.*;
import java.io.*;

class Solution{
    public static void main(String []argh){
        Scanner in = new Scanner(System.in);
        int n = in.nextInt();
        Map<String, Integer>mp = new HashMap<String, Integer>();
        for(int i = 0; i < n; i++){
            String name = in.next();
            int phone = in.nextInt();
            // Write code here
            mp.put(name, phone);
            
        }
        while(in.hasNext()){
            String s = in.next();
            // Write code here
            if (!mp.containsKey(s))
            System.out.println("Not found");
            else{
                System.out.println(s+"="+mp.get(s));
            }
        }
        in.close();
    }
}

File: 30 Days of Code/Python/Day8_Dictionary_Maps.py
# Enter your code here. Read input from STDIN. Print output to STDOUT

phoneBook = dict()
n = int(input())
for i in range(n):
    name, phone_number = input().split(" ")
    #print(name, phone_number)
    if i not in phoneBook:
        phoneBook[name] = phone_number
    
while True:
    try:
        name = input()
    except EOFError as e:
        break    
    if name in phoneBook:
            print(name+"="+phoneBook[name])
    else:
        print ("Not found")